src/ontology/cob-edit.owl

```diff
@@ -1347,6 +1347,15 @@
         <obo:IAO_0000115>A process during which an organism comes into contact with another entity.</obo:IAO_0000115>
         <rdfs:label xml:lang="en">exposure of organism</rdfs:label>
     </owl:Class>
+    
+    <!-- http://purl.obolibrary.org/obo/COB_0001002 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/COB_0001002">
+        <obo:IAO_0000115>A disease is a disposition to undergo pathological processes that exists in an organism because of one or more disorders in that organism.</obo:IAO_0000115>
```

Comment:
This seems to be used by OGMS, DOID, and Mondo.

Reply:
I would want the label to be: disease 
    and not to conflate disease and disorder in one label. 

Also the definition states that a disease is a result of a disorder.

Reply:
In OGMS, that is the definition for [disease](https://ontobee.org/ontology/OGMS?iri=http://purl.obolibrary.org/obo/OGMS_0000031), not "disease or disorder" [Disorder](https://ontobee.org/ontology/OGMS?iri=http://purl.obolibrary.org/obo/OGMS_0000045)  is a material entity in OGMS.

Reply:
the label disease is used in the following ontologies: 
Disease Ontology (DOID), EFO, OMIT, SIO, OGMS, ADO, AGRO, APOLLO_SV, CCO, CIDO, CTO, GENEPIO, GeXO, HTN, IDO, IDO-COVID-19, MFNO, MFOMD, OAE, OBI, OBIB, OHD, OHPI, OMIABIS, ONE, ONS, OPMI, PLANP, RO, ReTO, ReXO, VIDO, VO, Orphanet, GSSO, MI, BAO, ExO

Disease or Disorder 
   is used by: 
NCIT, MONDO

Reply:
Not to muddy the waters, but I have been thinking about redefining the DO's disease to be a biological process rather than a disposition.  Open to working a definition with the community, as it has been mentioned that there is interest in revising the current definition.